backend-es/src/Main.purs

```diff
@@ -377,24 +381,26 @@ esModulePackageJson = """{"type": "module"}"""
 
 spawnFromParentWithStdin :: String -> Array String -> Maybe String -> Aff Unit
 spawnFromParentWithStdin command args input = makeAff \k -> do
-  childProc <- ChildProcess.spawn command args defaultSpawnOptions
-    { stdio =
-        [ Just $ Pipe
-        , Just $ ShareStream (unsafeCoerce Process.stdout)
-        , Just $ ShareStream (unsafeCoerce Process.stderr)
+  -- To preserve colors in stdout need to pass it into spawn's stdio.
+  childProc <- unsafeCoerce <$> UnsafeChildProcess.spawn' command args
```

Comment:
Why is the unsafeCoerce needed here? Presumably the `map` is unnecessary?

Reply:
Because below I used `ChildProcess.stdin` and other functions for safe ChildProcess, if to use `unsafeStdin` it returns Nullable Writtable, it should be handled, map can be ommited yes.

Reply:
If you suppose it is more correct not ot use unsafeCoerce here, I can fix this too.

Reply:
Fixed.